models/domains_tlds.php

```diff
@@ -1446,9 +1453,12 @@ public function getDomainsCompanySettings($company_id = null)
      * @param array $tlds A list containing the TLDs to import from the registrar module
      * @param int $module_id The ID of the registrar module
      * @param int $company_id The ID of the company to import the TLDs
+     * @param array $filters A list of filters for the process
+     *
+     *  - terms A list of terms to import for the TLD, if supported
```

Comment:
The terms filter doesn't actually seem to be restricting the terms.  The steps I took were simply,

1. Select enom
2. Select only terms for 1, 2, and 3 years
3. Select a single TLD
4. Import
5. Examine pricing on the imported TLD
6. See that all 10 terms are imported